diagnostic.md

```diff
@@ -16,23 +16,21 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+/Users/grace/wdi/try
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
-_Replace this text with your answer._
+GitHub is a source code version control repository.
```

Comment:
A user interface (UI) for accessing git's version control capabilities for our projcet repositories.